src/chocolatey/StringResources.cs

```diff
@@ -82,6 +82,12 @@ public static class Package
                 /// <remarks>Business Edition Only Variable</remarks>
                 public const string ChocolateyLicenseType = nameof(ChocolateyLicenseType);
 
+                /// <summary>
+                /// The name or identifier of the package that is being handled.
+                /// </summary>
+                /// <seealso cref="ChocolateyPackageName" />

```

Comment:
```suggestion
                /// <seealso cref="ChocolateyPackageName" />
                /// <remarks>Available in v2.5.0+</remarks>
```

Would be a good idea to denote when it was added, so it flow through to our internal documentation.